data/campaigns/Under_the_Burning_Suns/units/quenoth/Tauroch_Protector.cfg

```diff
@@ -15,11 +15,11 @@
     {TRAIT_RESILIENT}
     {TRAIT_DEXTROUS}
     {TRAIT_HEALTHY}
-    hitpoints=71
+    hitpoints=59
```

Comment:
That's a huuuge nerf to HP, are you sure this is right?

Reply:
You are the one that asked me to do this because they felt too strong when you played them. x2

Reply:
Yeah but I seem to remember suggesting nerfing dmg more than HP. Unless I remembered wrong. Maybe we can make this at least 60 something? And if further dmg nerf would be needed to compensate then that's fine.